Readme update (#13)
neuronets/trained-models: README.md

```

```
datalad get -s osf-storage neuronets/braingen
```

## Loading models for training

> All models are available for re-training or transfer learning purposes except the **kwyk** model, which its weights are not available in tf2 keras format (We are working to make it available in near future). These models can be loaded with `tf.keras.models.load_model`.

Reviewer: ```suggestion
All models are available for re-training or transfer learning purposes except the **kwyk** model.  The kwyk model weights are not available in a tf2 keras format (We are working to make it available in near future). The kwyk models can be loaded with `tf.keras.models.load_model`.
```

```
import tensorflow as tf

model = tf.keras.models.load_model("neuronets/brainy/0.1.0/brain-extraction-unet-128iso-model.h5")
model.fit(...)
```

You can see a transfer learning example [here](https://github.com/neuronets/nobrainer/blob/master/guide/transfer_learning.ipynb), and an example of brain MRI generation using **braingen** models can be find [here](https://github.com/neuronets/nobrainer/blob/master/guide/train_generation_progressive.ipynb).

## Using models for inference

You can use [_Nobrainer_](https://github.com/neuronets/nobrainer) toolbox for inference.

```
import nobrainer

> from nobrainer.volume import standardize_numpy

Reviewer: i don't think this needs `standardize_numpy`. the tensorflow op should be sufficient.

```suggestion
from nobrainer.volume import standardize
```
